HousingSearchListener.Tests/AwsMockApplicationFactory.cs

```diff
@@ -0,0 +1,28 @@
+﻿using HousingSearchListener.V1.Infrastructure;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+using Nest;
+
+namespace HousingSearchListener.Tests
+{
+    public class AwsMockApplicationFactory
```

Comment:
Well it's just a mock container used to host the service so that integration tsets will run. In this case it uses ES, but other services may use Aws.
I'll remove the "Aws" from the name so that it is just `MockApplicationFactory`.